pypeit/wavecalib.py

```diff
@@ -16,7 +16,8 @@
 from pypeit.core import trace_slits
 
 from pypeit import debugger
-from IPython import embed
+import IPython
```

Comment:
Is there a reason to do this?  Do we need to import all of IPython?

Reply:
I'm unclear on what our convention is for this. We only ever use embed. However, for things like interpolation, or other modules, we usually import scipy, etc. Should the rule be if you are only using one function from a module just import that function.

Reply:
The only thing I'm against is using 'as' (except for `np` and `plt` because I lost that battle before even starting to code in python).  Other than that, I use full imports vs. from imports on a case by case basis.  Depending on how well the package is setup, it can be a lot faster to do `from ...`.  For `scipy`, I always use, e.g., `from scipy import interpolate`.  In this case, when you're only ever using one function, I expect it's more efficient to use `from IPython import embed`.

Reply:
Okay will do, will slowly change those back.